src/components/Search.js

```diff
@@ -69,6 +75,11 @@ export const Search = () => {
   return (
     <div class='search'>
       <img class='double-u' src='images/w.svg' style={{ display: ((searchResults || !isOnline) ? 'none' : 'block') }} />
+      <div class='home-text-box' style={{ display: ((searchResults || !isOnline) ? 'none' : 'block') }}>
+        <div class='welcome-text'>
+          {i18n.i18n('welcome-search-text')}
+        </div>
+      </div>
       <input ref={inputRef} type='text' placeholder={i18n.i18n('search-placeholder')} value={query} onInput={onInput} data-selectable />
```

Comment:
the placeholder should be in search language.

Reply:
Confirming with design: https://phabricator.wikimedia.org/T248080#5993467